m34thook/scanner_core.cpp

```diff
@@ -295,17 +295,17 @@ namespace initial_scanners {
 		sqrt_locator_entry,
 		//locate_readfile_entry,
 	//	locate_level_reload_entry,
-		locate_init_func_rva_992170_entry//,
-		//idgamesystemlocal_locator_entry
+		locate_init_func_rva_992170_entry,
+		idgamesystemlocal_locator_entry
```

Comment:
This is great, thanks for the contribution. I had already added these modifications to the changes here: https://github.com/FishnCrisps/m3337ho0o0ok/commit/10dee47a015532961bd5a3c81ffa56fc2b47fac6
It also fixes issues with new code that was untested.

Since you took on making modifications to the gamesystemlocal locator, it would be awesome if you could merge the two scanners, so idgamesystemlocal_locator_entry is unnecessary.